catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -105,23 +111,25 @@ public CreateStorageResponse create(CreateStorageRequest createRequest)
                 .size());
 
         for (ContentItem contentItem : contentItems) {
-
-            String id = contentItem.getId();
-
-            Path contentIdDir = Paths.get(baseContentTmpDirectory.toAbsolutePath()
-                    .toString(), createRequest.getId(), id);
-
             try {
+                ContentItemValidator.validate(contentItem);
```

Comment:
This throws a runtime exception that you'll just let bubble up. I've no problem with that in general, but it seems like that case is similar to the one where a `URISyntaxException` was thrown and that one is being caught and wrapped in a `StorageException`. Might make more sense to do the same with this exception to be consistent?